pyvelox/pyvelox.h

```diff
@@ -69,6 +69,50 @@ inline velox::variant pyToVariant(const py::handle& obj) {
   }
 }
 
+inline velox::variant pyToVariant(const py::handle& obj, const Type& dtype) {
+  if (obj.is_none()) {
+    return velox::variant();
```

Comment:
Even if obj is none, should we loose all type information ?

Reply:
We can create a null variant which has typekind information, we should use that imo.

Reply:
Tried returning a null variant with type information. Does this look good?